src/main/java/com/upserve/uppend/BlockedLongs.java

```diff
@@ -147,9 +147,7 @@ public synchronized void append(final long pos, final long val) {
 
     public LongStream values(long pos) {
         log.trace("streaming values from {} at {}", file, pos);
-        if (outDirty.get()) {
```

Comment:
Looks like these are the only two gets of `outDirty` - can it be removed?

Reply:
Good catch